autoinstrumentation/python/requirements.txt

```diff
@@ -0,0 +1,43 @@
+opentelemetry-distro==0.25b2
+# We don't use the distro[otlp] option which automatically includes exporters since gRPC is not appropriate for
```

Comment:
I made this decision, but not sure on it. @owais In particular perhaps you have some advice on what we should use here in our "autoinstrumentation distro". The somewhat annoying aspect is that the operator will have to specify `OTEL_TRACES_EXPORTER` for the user.

Reply:
I think using proto http sounds reasonable if you can get away with it. I was working on a similar feature at some point and had decided to build one image per platform/architecture.